sql/updates/world/2015_09_14_00_world.sql

```diff
@@ -0,0 +1,82 @@
+-- creature (2 entries)
+SET @CGUID = 1000000;
```

Comment:
I told him to use such huge guid to avoid travis blaming because SET @CGUID = XXXX; and be sure it don't uses existing guids :P ofc when merged it must be changed to unused guids on range 1-213000.